lib/index.ts

```diff
@@ -0,0 +1,405 @@
+/**
+ * @packageDocumentation
+ * TopStats - Official Node.js client for the topstats.gg API
+ * @version 1.0.0
+ */
+
+/**
+ * Response structure for rate limit errors
+ */
+export interface RateLimitedResponse {
```

Comment:
Having everything be in one file turns to a bit much. Why not have a types.ts file? (And a Client.ts file)

Reply:
This is only temporary while I lay the groundwork and wait for the API to closer to being finished